app/src/main/java/org/resolvetosavelives/red/bp/BloodPressureCategory.kt

```diff
@@ -0,0 +1,77 @@
+package org.resolvetosavelives.red.bp
+
+import org.resolvetosavelives.red.R
+import org.resolvetosavelives.red.util.Just
+import org.resolvetosavelives.red.util.None
+import org.resolvetosavelives.red.util.Optional
+
+enum class BloodPressureCategory(private val urgency: Int, val displayTextRes: Optional<Int>) {
+
+  EXTREMELY_HIGH
+      (4, Just(R.string.bloodpressure_category_extremely_high)),
+
+  VERY_HIGH
+      (3, Just(R.string.bloodpressure_category_very_high)),
+
+  MODERATELY_HIGH
+      (2, Just(R.string.bloodpressure_category_moderately_high)),
+
+  MILDLY_HIGH
+      (1, None),
+
+  NORMAL
+      (0, None),
+
+  LOW
+      (-1, Just(R.string.bloodpressure_category_low));
+
+  fun isUrgent(): Boolean {
```

Comment:
I'm open for a better name than `isUrgent()`. This method is used for deciding if a BP reading should be shown in red on the summary screen.

Reply:
Tagging @deobald because he's def. better than us at naming things.